test/views/TransportViewSpec.scala

```diff
@@ -16,102 +16,90 @@
 
 package views
 
-import forms.Choice.{Arrival, Departure}
+import controllers.routes
 import forms.Transport
 import helpers.views.{CommonMessages, TransportMessages}
 import play.api.data.Form
+import play.api.i18n.MessagesApi
 import play.twirl.api.Html
-import views.spec.ViewSpec
+import utils.Injector
+import views.html.transport
+import views.spec.UnitViewSpec
+import views.spec.UnitViewSpec.instanceOf
 
-class TransportViewSpec extends ViewSpec with TransportMessages with CommonMessages {
+class TransportViewSpec extends UnitViewSpec with TransportMessages with CommonMessages with Injector {
```

Comment:
Why we would like to have injector here ?

Reply:
Good point, I'll remove it